charts/wildfly-common/templates/_ingress.yaml

```diff
@@ -0,0 +1,30 @@
+{{- define "wildfly-common.ingress.tpl" -}}
+kind: Ingress
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: {{ include "wildfly-common.appName" . }}
+  labels: {}
+  annotations: {}
+spec:
+  ingressClassName: nginx
```

Comment:
should the class of the ingress be configurable?  Or should it be confused to use the default class?

Reply:
I wasn't sure how often the default class is defined or if we should set it

Reply:
Maybe if the value is not set we don't add it